tests/test_delete_account.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from flask import url_for
+from flask.testing import FlaskClient
+
+from hushline.db import db
+from hushline.model import (
+    User,
+)
+
+
+@pytest.mark.usefixtures("_authenticated_user")
+def test_delete_account(client: FlaskClient, user: User) -> None:
```

Comment:
Being able to delete the last user even when not an admin feels wrong. Is this the behavior we want?